tools/glcli/pyproject.toml

```diff
@@ -14,13 +14,14 @@ python = "^3.8"
 click = "^8"
 grpcio = "^1"
 protobuf = "^5"
-gl-client = { path = "../../libs/gl-client-py", develop = false}
+gl-client = "0.2.0"
```

Comment:
Are we going to publish `glcli` with `0.2`? If it is not already published I would rather prefer to postpone a cli tool release until we have a rust version of `glcli` and keep it as a developer tool until then.